docs/content/users/basics/troubleshooting.md

```diff
@@ -128,86 +125,101 @@ On Windows CMD, use [sysinternals tcpview](https://docs.microsoft.com/en-us/sysi
-* Missing index.php: There may not be an index.php or index.html in your project.
+* Misconfigured docroot: If the docroot isn’t where the web server thinks it is, then the web server won’t find `index.php`. Look at your `.ddev/config.yaml` to verify it has a docroot containing `index.php`. It should be a relative path.
+* Missing `index.php`: There may not be an `index.php` or `index.html` in your project.
 
-## `ddev start` fails and logs contain "failed (28: No space left on device)" - Docker File Space
+## `ddev start` Fails and Logs Contain "failed (28: No space left on device)" - Docker File Space
 
-If `ddev start` fails, it's most often because the web container or db container fails to start. In this case the error message from `ddev start` says something like "Failed to start <project>: db container failed: log=, err=container exited, please use 'ddev logs -s db' to find out why it failed". You can`ddev logs -s db` to find out what happened.
+If `ddev start` fails, it’s most often because the `web` or `db` container fails to start. In this case, the error message from `ddev start` says something like “Failed to start <project>: db container failed: log=, err=container exited, please use 'ddev logs -s db' to find out why it failed”. You can`ddev logs -s db` to find out what happened.
 
-If you see any variant of "no space left on device" in the logs when using Docker Desktop, it means that you have to increase or clean up Docker's file space. To increase the allocated space, go to "Resources" in Docker's Preferences as shown in ![Docker disk space](../images/docker_disk_space.png)
+If you see any variant of “no space left on device” in the logs when using Docker Desktop, it means you have to increase or clean up Docker’s file space. Increase the “Disk image size” setting under “Resources” in Docker’s Preferences: ![Docker disk space](../images/docker_disk_space.png).
 
-If you see "no space left on device" on Linux, it most likely means your filesystem is full.
+If you see “no space left on device” on Linux, it most likely means your filesystem is full.
 
-## `ddev start` fails with "container failed to become ready"
+## `ddev start` Fails with "container failed to become ready"
 
-If a container fails to become ready, it means it's failing the healthcheck.  This can happen to any of the containers, but you can usually find out quickly what the issue is with a `docker inspect` command.
+A container fails to become ready when its health check is failing. This can happen to any of the containers, and you can usually find the issue with a `docker inspect` command.
 
-You may need to install [jq](https://stedolan.github.io/jq/download/) for these, `brew install jq`, or just remove the "| jq" from the command and read the raw json output.
+!!!tip
```

Comment:
I didn't understand why you sometimes use a bare "tip" and other times put "what the tip is about" after it.

Reply:
Efficiency. If I could convey the content of the tip in a short title or phrase, I’d use the tip heading and otherwise leave it bare. Two reasons:

1. **Balance.** A *long*, bold heading is more disruptive to the page flow. Tips already get visual emphasis, so a wordy bold heading pushes the whole thing too far away from *helpful aside* and into the realm of *visual assault*.
2. **Skimming.** The heading can help me know whether to read or skip the supporting text, which is vital if I’m scanning for information I need.